vernac/lemmas.ml

```diff
@@ -327,8 +327,8 @@ let universe_proof_terminator compute_guard hook =
   | Proved (opaque,idopt,proof) ->
       let is_opaque, export_seff, exports = match opaque with
```

Comment:
Exports is now always `[]` and hence `check_exports` down below is dead code.

Reply:
Well spotted, grazie; fixed.